improver/cli/phase_change_level.py

```diff
@@ -62,6 +67,10 @@ def process(*cubes: cli.inputcube, phase_change, horizontal_interpolation=True):
 
                 snow-sleet - the melting of snow to sleet.
                 sleet-rain - the melting of sleet to rain.
+        grid_point_radius (int):
+            The radius in grid points used to calculate the maximum
+            height of the orography in a neighbourhood as part of this
+            calculation.
```

Comment:
It might be worth adding a sentence to explain the handling if `grid_point_radius` is zero, otherwise, someone might assume setting `grid_point_radius` to zero might raise an error. Something like:
```suggestion
        grid_point_radius (int):
            The radius in grid points used to calculate the maximum
            height of the orography in a neighbourhood as part of this
            calculation. If this is zero, the orography is used directly 
            without finding the maximum height in a neighbourhood.
```
Note that the docstring in the `__init__` of the `PhaseChangeLevel` plugin should also be updated to match.

Reply:
Done